packages/integration-test-app/package.json

```diff
@@ -11,9 +11,9 @@
   },
   "dependencies": {
     "chai": "^4.2.0",
-    "node-rnw-rpc": "^1.0.5-24",
-    "react": "17.0.1",
-    "react-native": "0.0.0-d04ec4dc6",
+    "node-rnw-rpc": "^1.0.5-23",
```

Comment:
Do you know how this became lower?

Reply:
Probably a merge error after resolving the conflict due to nightly RNW builds

Reply:
@chiaramooney Remember to correct this here and other package.json files